src/reviews/service.py

```
@@ -0,0 +1,76 @@
+            new_review.user = user
+            new_review.book = book
+            session.add(new_review)
+            await session.commit()
+            return new_review
+
+        except Exception as e:
+            logging.exception(e)
+            raise HTTPException(
+                detail="Oops... somethig went wrong!",
+                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)